src/mudlet.cpp

```diff
@@ -1668,61 +1666,12 @@ Host * mudlet::getActiveHost()
         return 0;
 }
 
-
-
 void mudlet::addSubWindow( TConsole* pConsole )
 {
     mainPane->layout()->addWidget( pConsole );
     pConsole->show();//NOTE: this is important for Apple OSX otherwise the console isnt displayed
 }
 
-void mudlet::closeEvent(QCloseEvent *event)
```

Comment:
You specify override on the declaration, which it is for this one. I just didn't notice it when I deleted it.